.github/workflows/shellcheck.yml

```diff
@@ -0,0 +1,18 @@
+name: ShellCheck
+
+on:
+  pull_request:
+    branches:
+      - main
+  push:
+    branches:
+      - main
+
+jobs:
+  shellcheck:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Run shellcheck
```

Comment:
What about adding these two lines to the lint job to reduce complexity in the workflows?

https://github.com/ChainSafe/forest/blob/main/.github/workflows/rust.yml#L100

Reply:
I agree, however, renaming it means we need to mess with all the PRs again. Maybe just keep it in `shell.yml` for now.

Reply:
Sounds good to me!